change/@fluentui-cra-template-f19ad23e-0c38-418c-b102-834d12de9304.json

```diff
@@ -1,6 +1,6 @@
 {
   "type": "patch",
-  "comment": "Temporarily add resolution for `@babel/core` to work around bug",
+  "comment": "Revert adding resolution for babel because bug is fixed",
```

Comment:
Note, this is appearing as a rename because it's deleting the old change file and adding a new one. If this goes in before the publish build, the change type here can be changed to `none`.